lib/assets/javascripts/cartodb/common/dialogs/delete_column/delete_column_view.js

```diff
@@ -24,9 +24,8 @@ module.exports = BaseDialog.extend({
   },
 
   _initViews: function() {
-
     this.table = this.options.table;
-    this.column = this.options.column;
+    this.column = this.options.table.get('column_aliases')[this.options.column] || this.options.column;
```

Comment:
I believe that 

```js
undefined[<whatever>]
```
 will create an error. Can you modify this so it's resilient to `null` or `undefined` `this.options.table.get('column_aliases')` values?